PlanningPoker/Services/IUserService.cs

```diff
@@ -8,21 +8,20 @@ namespace PlanningPoker.Services
 {
     public interface IUserService
```

Comment:
It seems to me, that this interface violates Interface segregation principal.

Reply:
Task<Room> AddRoom(Room room);
         void DeleteRoom(string id);
        Task<List<Room>> GetRooms();

Looks like this are not related to users